bitdefender/assets/logs/bitdefender_tests.yaml

```diff
@@ -0,0 +1,138 @@
+id: bitdefender
+tests:
+ -
+  sample: |-
+    {
+      "method" : "addEvents",
```

Comment:
Are there any other different methods you could test with? If all events have the same format, no worries.

Reply:
This is the only method available that pushes the data i.e. add_events